docs/UserGuide.md

```diff
@@ -132,7 +132,7 @@ Users can:
    Our provided sample data is not accurate nor necessary to the application. As such, you can easily delete the sample data in the following way:
    
    * Usage of the `clear` command.<br>
-     Clears all flashcard and deck data currently in FlashNotes. Refer under [Main Mode Features](#Main Mode Features) for further details about the command.
```

Comment:
Actually the correct format for in-document links are (#main-mode-features) and (#faq) which was probably why the links did not work.